examples/tools.mk

```diff
@@ -0,0 +1,36 @@
+
+ifneq ($(USE_DOCKER),)
+# Use Docker tools
+
+DOCKER=docker
+#DOCKER=podman
+
+PWD = $(shell pwd)
+DOCKERARGS = run --rm -v $(PWD)/../..:/src \
+	-w /src/examples/$(notdir $(PWD))
+
+GHDL      = $(DOCKER) $(DOCKERARGS) ghdl/synth:beta ghdl
+GHDLSYNTH = ghdl
+YOSYS     = $(DOCKER) $(DOCKERARGS) ghdl/synth:beta yosys
+NEXTPNR   = $(DOCKER) $(DOCKERARGS) ghdl/synth:nextpnr-$(PLATFORM) nextpnr-$(PLATFORM)
+ECPPACK   = $(DOCKER) $(DOCKERARGS) ghdl/synth:trellis ecppack
+ICEPACK   = $(DOCKER) $(DOCKERARGS) ghdl/synth:trellis icepack
+OPENOCD   = $(DOCKER) $(DOCKERARGS) --device /dev/bus/usb ghdl/synth:prog openocd
+
+ICEPORG   = # ?
+ICETIME   = # ?
```

Comment:
Before merging, could someone check if `iceprog` and `icetime` exist in the docker image?
And if they do, how would they be included and would they need any flags like `openocd` does?